src/game-of-life/time-profiling.md

```diff
@@ -74,7 +75,29 @@ max of last 100 = ${Math.round(max)}
 `.trim();
   }
 };
+export {fps};
 ```
+In `index.js` add at the top level :
+```js
+import {fps} from "./fps";
+```
+Now in `renderLoop` we update our (FPS):
+```js
+const renderLoop = () => {
+    fps.render(); //new
+    
+    universe.tick(); 
+    drawCells();
+    drawGrid();
+
+    animationId=requestAnimationFrame(renderLoop);
+};
+```
+Dont forget to Add the `fps` element to `index.html` : 
+```html
+    <div id="fps"><div>
+```
+And Voila! Now you have an FPS counter !
```

Comment:
Nitpick: "counter !" -> "counter!"

Reply:
Looks like this one didn't get fixed -- other than that this is perfect! Will merge after this last fixup. Thanks again!

Reply:
Done , plus I added @mgattozzi  instructions , don't know why I got a -> b wrong , taught that that means you already have changed it and you're just telling me xD